extensions.json

```
@@ -1343,6 +1343,10 @@
     {
       "id": "shyykoserhiy.git-autoconfig",
       "repository": "https://github.com/ShyykoSerhiy/git-autoconfig"
+    },
+    {
+      "id": "ahmadawais.emoji-log",
+      "repository": "https://github.com/ahmadawais/Emoji-Log"
```

I see that this repository publishes official release tags.

When that's the case, we prefer to pin a specific version & publishing from the corresponding release tag, as opposed to just publishing from the default branch (may contain WIP changes).

```suggestion
      "repository": "https://github.com/ahmadawais/Emoji-Log",
      "version": "0.7.0",
      "checkout": "v0.7.0"
```

---

Awesome! thank you.